### PR TITLE
Add press logos and FT link

### PR DESCRIPTION
**Pull request by fffw**
Resolves https://github.com/getlantern/lantern/issues/2579

### CODE REVIEW

File: app/styles/_top.scss [hunk 1]
```diff
@@ -217,18 +215,17 @@ div.outer-container {
...
                           text-align: center;
                           text-transform: uppercase;
                           padding-top: 20px;
-                          display: inline-block;
-                          @media screen and (max-width: $break-small) {
+                          @media screen and (max-width: $break-medium) {
                               padding-bottom: 30px;
                               padding-top: 15px;
-                              background-image: url("./images/nav-bg.png");
+                              background-image: url("/images/nav-bg.png");
```
Comment (fffw) [inline app/styles/_top.scss]: Removing the dot will make `grunt serve` show all images. I updated `deploytos3.bash` accordingly so hopefully it won't affect in production.

**Comment by fffw** [comment]
The header nav is centered now. It doesn't look good on certain breakpoints, but not worse than current site.

**Comment by fffw** [comment]
@atavism Do you have time to take a look? I just forwarded the email discussing about design to you. Thanks in advance!

**Comment by atavism** [comment]
@fffw Sure. I will look at this shortly.

**Comment by atavism** [comment]
This looks good to me, @fffw. I think the image path rewrites are fine (I tested locally). Let's wait until the new version of FT is ready to deploy this to production since the link will show up on the main page. Will merge this for now though.

**Comment by fffw** [comment]
Thanks for reviewing @atavism. I think the FT link opens a new tab to
firetweet.io rather than downloads the APK directly. As you are busy on FT,
I can push the site into production.

**Pull request merged**